libcloud/compute/drivers/equinixmetal.py

```diff
@@ -701,246 +703,14 @@ def ex_disassociate_address(self, address_uuid, include=None):
         result = self.connection.request(path, params=params, method="DELETE").object
         return result
 
-    def list_volumes(self, ex_project_id=None):
```

Comment:
I can confirm that the Storage APIs have been sunset since June 2021. There is no direct alternative to this service.